src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -673,6 +682,7 @@ public DarCollection createElectionsForDarCollection(User user, DarCollection co
     } catch (Exception e) {
       logException("Exception creating elections and votes for collection: %s".formatted(
           collection.getDarCollectionId()), e);
+      throw e;
```

Comment:
This is a change in API behavior, is that part of this ticket? Was something broken by not throwing an exception which is now fixed?

Reply:
Yes, the error wasn't being percolated up to the caller and it continued on and called `darCollectionDAO.findDARCollectionByCollectionId`, which is invalid in this case.

Reply:
OK, I think see. What I'd expect to see here is a catch that rethrows as a more specific exception and wrap the previous one as a cause. The log message would instead be the new exception's message. And finally if we wanted to log, we could do it in the top level catch handler.

But it looks like that's not how the consent service works right now, so I agree it's best to keep with the current process.

I do think nesting `try`s isn't great though, and now that this method `throws Exception` you no longer need the outer `catch`